.travis.yml

```diff
@@ -9,10 +9,7 @@ before_install:
   - docker-compose -v
 
 install:
-  - docker-compose run --rm drivers-download composer install
-  - docker-compose run --rm drivers-download php driver/downloadDriver.php
-  - docker-compose build app
-  - docker-compose run --rm app
+  - docker-compose run --rm app composer ci
```

Comment:
ok, I'm ok with moving it here from the docker-compose command property.  I agree that it is more clear.